x-pack/plugins/uptime/public/components/monitor/ml/use_anomaly_alert.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useContext, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getExistingAlertAction } from '../../../state/actions/alerts';
+import { alertSelector } from '../../../state/selectors';
+import { UptimeRefreshContext } from '../../../contexts';
+import { useMonitorId } from '../../../hooks';
+
+export const useAnomalyAlert = () => {
```

Comment:
Presumably something in here needs to change to set the state correctly immediately after alert creation (without a page refresh)

Reply:
fixed the issue